src/main/java/com/fundacionjala/geometry/Constants.java

```diff
@@ -0,0 +1,18 @@
+package com.fundacionjala.geometry;
+
+/**
+ * Class to define constants used in several classes
+ *
+ * @author Rosario Garcia
+ */
+public class Constants {
```

Comment:
Don't forget to add a private constructor for constant class (static methods and attributes)